src/pixelator/test_utils/report.py

```diff
@@ -59,6 +60,7 @@ def test_report_run(self):
 
     @pytest.mark.dependency(depends=["test_report_run"])
     def test_report_customer_exists(self):
-        report_files = (self.workdir / "report").glob("*_report.html")
+        """Check if the qc report html exists."""
+        report_files = (self.workdir / "report").glob("*.qc-report.html")
```

Comment:
I don't understand how this cannot be tested if L65 and L66 are tested. Are we not testing the function at all?

Reply:
Good catch, this test is indeed useless as is. I have updated it.